build-config/scripts/onie-mk-demo.sh

```diff
@@ -25,6 +25,10 @@ if [ "$arch" = "powerpc-softfloat" -o "$arch" = "armv7a" ] ; then
     # Both of these architectures share common installer code as they
     # are both based on u-boot.
     arch_dir="u-boot-arch"
+elif [ "$arch" = "x86_64" -o "$arch" = "armv8a" ] ; then
+    # These architectures share common installer code as they
+    # are based on grub.
+    arch_dir="grub-arch"
```

Comment:
@david56 , @pankajbansal3073 -- well this is a fundamental issue to me.  Either armv8a (aarch64) is going to be GRUB or it will be U-Boot.  It cannot be either.

powerpc -- is only U-BOOT
x86 -- is only GRUB (though could be "bios" or "uefi")

This is a big topic and I will raise the awareness on the mailing list shortly.  I only want ONIE to support UEFI/GRUB for aarch64.  That is the modern approach, what servers are doing.